### PR TITLE
[86014] FSR Feature Flag Removal (vets-website)

### PR DESCRIPTION
**Pull request by maxx1128**
## Summary

Three feature flags have been set to `true` by default and there are no longer any in-progress forms that have them set to `false`. This means the flags are safe to remove, and this pull request removes them on the `vets-website` side.

## Related issue(s)

- [Main ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/86014)
- [Initial ticket for the discovery](https://github.com/department-of-veterans-affairs/va.gov-team/issues/81813)

## Testing done

N/A. As long as all the existing tests pass, this is good.

## Screenshots

N/A

## What areas of the site does it impact?

FSR Forms, specifically in-progress

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [x]  Events are being sent to the appropriate logging solution
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by CBonade** [comment]
Platform review delayed due to checks not yet completed.

File: src/platform/utilities/feature-toggles/featureFlagNames.json [hunk 1]
```diff
@@ -34,7 +34,6 @@
   "coeAccess": "coe_access",
   "combinedDebtPortalAccess": "combined_debt_portal_access",
   "combinedFinancialStatusReport": "combined_financial_status_report",
```
Comment (Dr-Pongo) [inline src/platform/utilities/feature-toggles/featureFlagNames.json:36]: Hey I think `combinedFinancialStatusReport` can be removed as well!

**Review by Dr-Pongo — CHANGES_REQUESTED** [review]
Just one note on removing `combinedFinancialStatusReport` from `featureFlagNames.json`.

Edit:
Oh there's one more reference of combined in the following
`src/applications/mhv-secure-messaging/tests/e2e/fixtures/generalResponses/featureToggles.json`

**Comment by maxx1128** [comment]
Removed the extra flag and rebased off of `main` to also hopefully pass the tests.

**Review by Dr-Pongo — APPROVED** [review]
LGTM! 🚀

**Review by alex-morgun-1 — APPROVED** [review]

**Review by rsmithadhoc — APPROVED** [review]

**Pull request merged**